src/core/execution/DonateGoldExecution.ts

```diff
@@ -28,6 +32,7 @@ export class DonateGoldExecution implements Execution {
       this.sender.canDonate(this.recipient) &&
       this.sender.donateGold(this.recipient, this.gold)
     ) {
+      this.mg.stats().goldSend(this.sender, this.recipient, this.gold); // stats for donated gold
```

Comment:
Please put the comment above, because these lines are over the column limit with the comment, which will cause bad auto-formatting when we update our code formatting engine.
```suggestion
      // stats for donated gold
      this.mg.stats().goldSend(this.sender, this.recipient, this.gold);
```